doc/extension_store/README.md

```diff
@@ -0,0 +1,76 @@
+# Proposal for Extension Store Manifest
+
+Example for manifest: [manifest.jsonc](./manifest.jsonc)
+
+## Variable Replacements
+
+* Variables placeholders use the `"My variable 'variableName' has value '${variableName}'"` syntax
+* Naming conventions:
+  * User defined variables are in `lowerCamelCase`
```

Comment:
```suggestion
  * Variables prompted to users are written in `lowerCamelCase`
```
The term "user-defined" is misleading here, since it implies that users can add their own variables.